libs/camutils/src/Manipulator.cpp

```diff
@@ -144,6 +144,14 @@ Manipulator<FLOAT>::Builder& Manipulator<FLOAT>::Builder::raycastCallback(RayCal
     return *this;
 }
 
+
+
+template <typename FLOAT> typename
+Manipulator<FLOAT>::Builder& Manipulator<FLOAT>::Builder::panDisabled(bool enabled) {
+    details.panDisabled = enabled;
```

Comment:
this is *very* confusing. It should all be done in terms of "panning being enabled or not".